recipe/meta.yaml

```diff
@@ -40,6 +41,9 @@ outputs:
         - test -e $PREFIX/lib/libcryptominisat5.so.5.6       # [linux]
         - IF NOT EXIST %LIBRARY_BIN%\\libcryptominisat5.5.6.dll exit 1       # [win]
   - name: pycryptosat
+    build:
+      number: {{ build }}
+      skip: True  # [win and vc<14]
```

Comment:
@msarahan, without the `skip` here, `conda-smithy rerender` didn't work. `python` only had `('3.5', '3.6')` in the build matrix on Windows, which is correct. But the vectors for `vc` and the compiler were not shortened so that the `zip_keys` failed.

Plus I needed to add `build/number` here because otherwise some `assert` for requiring the number was hit.

Reply:
Sounds good.  Thanks for iterating on it.

Reply:
Put `build/number` into the outputs now because of
```
  File "/opt/conda/lib/python3.6/site-packages/conda_forge_ci_setup/build_utils.py", line 125, in make_build_number
    raise ValueError("More than one build number found, giving up")
ValueError: More than one build number found, giving up
```
but the linter doesn't seem to like it xD

Reply:
bummer.  We need to fix that in conda_forge_ci_setup.

Reply:
Oh, the `conda-forge-ci-setup` one is already fixed and I reverted that workaround here :wink:.
The other workaround with needing to provide `build/number` explicitly for the `pycryptosat` output is due to something in `conda-build`, IIRC.

Reply:
xref: https://github.com/conda/conda-build/pull/3175